src/main/java/Storage.java

```diff
@@ -0,0 +1,68 @@
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.ArrayList;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class Storage {
+
+    private String filePath;
+    private String SPLIT =  " \\| ";
+    private String JOIN = " | ";
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
Consider using the `List` ADT instead (i.e., `List<Tast> taskList = new ArrayList<>()`).